sweetalert2.d.ts

```diff
@@ -302,27 +302,27 @@ declare module 'sweetalert2' {
      *
      * @param paramName The parameter to check
      */
-    function isValidParameter(paramName: string): boolean;
+    function isValidParameter(paramName: keyof SweetAlertOptions): boolean;
 
     /**
      * Determines if a given parameter name is valid for `Swal.update()` method.
      *
      * @param paramName The parameter to check
      */
-    function isUpdatableParameter(paramName: string): boolean;
+    function isUpdatableParameter(paramName: UpdatableParameters): boolean;
 
     /**
      * Normalizes the arguments you can give to Swal.fire() in an object of type SweetAlertOptions.
      *
      * Example:
      * ```
      * Swal.argsToParams(['title', 'text']); //=> { title: 'title', text: 'text' }
-     * Swal.argsToParams({ title: 'title', text: 'text' }); //=> { title: 'title', text: 'text' }
+     * Swal.argsToParams([{ title: 'title', text: 'text' }]); //=> { title: 'title', text: 'text' }
```

Comment:
I guess it was wrong. Can you confirm it, @limonte?

Reply:
Confirming :+1: